src/WebJobs.Extensions.DurableTask/Listener/TaskOrchestrationShim.cs

```diff
@@ -167,7 +167,8 @@ public override async Task<string> Execute(OrchestrationContext innerContext, st
             }
             catch (Exception e)
             {
-                if (orchestratorInfo.IsOutOfProc
+                if (orchestratorInfo != null

```

Comment:
Just to clarify, we still run in this code path when an orchestrator doesn't exist, so this makes sure we do not hit a NullReferenceException?

Reply:
Yes, exactly!